README.md

```diff
@@ -79,12 +90,120 @@ const {
   positionals
 } = parseArgs({ args, options });
 console.log(values, positionals);
-// Prints: [Object: null prototype] { foo: true, bar: 'b' } []ss
+// Prints: [Object: null prototype] { foo: true, bar: 'b' } []
 ```
 
 `util.parseArgs` is experimental and behavior may change. Join the
 conversation in [pkgjs/parseargs][] to contribute to the design.
 
+### `parseArgs` `tokens`
+
+Detailed parse information is available for adding custom behaviours by
+specifying `tokens: true` in the configuration.
+The returned tokens have properties describing:
+
+* all tokens
+  * `kind` {string} One of 'option', 'positional', or 'option-terminator'.
+  * `index` {number} Index of element in `args` containing token. So the
+    source argument for a token is `args[token.index]`.
+* option tokens
```

Comment:
To improve the readability I would explain that we mean the `kind=option` token

```suggestion
* 'option' tokens
```

Reply:
rather than quotes, which i find confusing, how about linking the term to the appropriate section?

Reply:
Yeah, that works too.

I use the same syntax in the `kind` description to be homogeneous

Reply:
The compact POJO description is a bit subtle to read. How about an expanded version with all the properties listed?

## Proposed expanded

A returned token has two properties which are always defined,
and some other properties which vary depending on the `kind`:
  * `kind` {string} One of 'option', 'positional', or 'option-terminator'.
  * `index` {number} Index of element in `args` containing token. So the
    source argument for a token is `args[token.index]`.

An option token has additional parse details
for an option detected in the input `args`:
  * `kind` = `'option'`
  * `index` {number} Index of element in `args` containing token.
  * `name` {string} Long name of option.
  * `rawName` {string} How option used in args, like `-f` of `--foo`.
  * `value` {string | undefined} Option value specified in args.
    Undefined for boolean options.
  * `inlineValue` {boolean | undefined} Whether option value specified inline,
    like `--foo=bar`.

A positional token has just one additional property with the positional value:
  * `kind` = `'positional'`
  * `index` {number} Index of element in `args` containing token.
  * `value` {string} The value of the positional argument in args (i.e. `args[index]`).

An option-terminator token has only the base properties:
  * `kind` = `'option-terminator'`
  * `index` {number} Index of element in `args` containing token.

## Old compact

The returned tokens have properties describing:

* all tokens
  * `kind` {string} One of 'option', 'positional', or 'option-terminator'.
  * `index` {number} Index of element in `args` containing token. So the
    source argument for a token is `args[token.index]`.
* option tokens
  * `name` {string} Long name of option.
  * `rawName` {string} How option used in args, like `-f` of `--foo`.
  * `value` {string | undefined} Option value specified in args.
    Undefined for boolean options.
  * `inlineValue` {boolean | undefined} Whether option value specified inline,
    like `--foo=bar`.
* positional tokens
  * `value` {string} The value of the positional argument in args (i.e. `args[index]`).
* option-terminator token
```